### DISCUSSION TITLE
Front-End Developer

### DISCUSSION THREAD

**Discussion by fahriakbaba** [community/community#30263] [Other Feature Feedback, Questions, & Ideas]
Is SVG in HTML  important for Front-End Developer?

**Comment by radar-blip** [comment]
I would think so. Depends on what you're working on though. If you're creating websites, you might want to use svg images for buttons and other graphics.

**Comment by taliba570** [comment]
In this modern world no webpage will look good without an icon or an image in most cases, and as the name suggests Front-End Developer, it is always advised to keep your arsenal full so you never fall short. In your scheduled learning time look around and learn how to wrangle SVGs.

Keep learning. 👍🏼

**Comment by als-creator** [comment]
yes, this is important, you need to be able to use design tools, illustrator, figma, inkscape, at least a little to do simple things

**Comment by lsprdev** [comment]
Yes, it is, if you what to learn more about it I recommend this little course https://css-tricks.com/lodge/svg/ it helped me a lot
